templates/apollo-graphql-react/files/back/src/resolvers/query.js

```diff
@@ -0,0 +1,55 @@
+import fetch from "node-fetch";
+
+const {
+  YOUTUBE_API_KEY = "<EXPORT YOUTUBE_API_KEY WITH YOUR KEY>",
+} = process.env;
+
+export const Query = {
+  deployFridayEpisodes(
+    _parent,
+    _args,
+    { dataSources: { additionalEpisodeInfo } }
+  ) {
+    return fetch(
+      `https://youtube.googleapis.com/youtube/v3/playlistItems?maxResults=100&part=snippet&playlistId=PLn5EpEMtxTCmLsbLgaN3djvEkRdp-YmlE&key=${YOUTUBE_API_KEY}`
```

Comment:
Shouldn't this URL, or at least the playlist ID, be a variable or config value somewhere?  Hard coding it seems wrong.

Reply:
Yeah, that'd be better. Since it's just a demo app and to keep things straightforward, there's a bunch of things I haven't done that I would normally do (e.g. proper error handling, a few abstraction layers, ...).
I also started by having a dynamic playlist ID, sent as a query variable by the front-end so any playlist could be fetched but I gave up on this idea, as it comes with other things to handle that would — I think — just add noise.

I guess it's a matter of where we want to set the good-practices cursor for targeted demos?
I will put the playlist ID in a const alongside the API key (but keep the value hard-coded). Going more into The Right Way would add more files and indirections that I think are not worth it for the sake of this example.

Reply:
Done in 2ee8ce95c5b449567c76267bef0ca14e706460dd
Let me know if you're not okay with this :)